src/PKSim.CLI.Core/Services/BatchConverterRunner.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using OSPSuite.Core.Commands.Core;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Services;
+using OSPSuite.Utility;
+using OSPSuite.Utility.Extensions;
+using PKSim.CLI.Core.RunOptions;
+using PKSim.Core;
+using PKSim.Core.Batch;
+using PKSim.Core.Commands;
+using PKSim.Core.Extensions;
+using PKSim.Core.Model;
+using PKSim.Core.Services;
+using PKSim.Core.Snapshots.Services;
+using Simulation = PKSim.Core.Batch.Simulation;
+
+namespace PKSim.CLI.Core.Services
+{
+   internal class BatchConverterRunner : IBatchRunner<BatchConverterRunOptions>
```

Comment:
This is the CLI command to convert batch to snapshot. This can be deleted once we are happy with the conversion